nvflare/job_config/fed_job_config.py

```diff
@@ -148,12 +148,14 @@ def simulator_run(self, workspace, clients=None, n_clients=None, threads=None, g
                     + workspace
                 )
                 if clients:
+                    clients = self._trim_whitespace(clients)
```

Comment:
Can this be done simply by: clients.strip()?

Reply:
no, simple strip() does not trim the white space in the middle, for example: "site-0,    site-1", the space in the middle won't be removed.